Sprint-1/exercises/random.js

```diff
@@ -2,8 +2,12 @@ const minimum = 1;
 const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+/*due to rules of precedence, parenthesis executes first. Inside parenthesis, subtraction executes first, 
+then addition executes.*/
```

Comment:
Can you explain what each part of the the expression is trying to calculate?
For examples,
- What does the expression `maximum - minimum + 1` try to calculate (in lament terms)? Why add one to the difference between `maximum` and `mininum`?
- What are the possible random values yielded by this expression: `Math.random() * (maximum - minimum + 1)`?
- What are the possible random values assigned to `num`? ( ? <= num <=   ?)

If we changed the value of `minimum` and `maximum` to -5 and 5 respectively, what are the possible random values assigned to num?